packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -182,7 +176,7 @@ const VIcon = mixins(
     const icon = this.getIcon()
 
     if (typeof icon === 'string') {
-      if (icon.indexOf('svg-') === 0) {
+      if (icon[0] === 'M' && icon[icon.length - 1] === 'Z') {
```

Comment:
`startsWith`/`endsWith`

Reply:
svg path's aren't guaranteed to have this structure, so it'll only work for mdi

Reply:
> svg paths also aren't guaranteed to have this structure, so it'll only work for mdi

true, but the current `svg-` prefix also only works for `@mdi/js` 🤷‍♂️

Reply:
The previously `svg-` prefix was because we shouldn't assume that every icon in every css icon library doesn't starts with `M` and doesn't ends with `Z`.
It was working with any other svg path if you prefix it with `svg-`, not only `@mdi/js`.

Reply:
While I get that and agree, is there another icon library that exports its icons as svg, and specifically a svg `d` path-only and with a 24px viewport? I just noticed that the current implementation already heavily targets `@mdi/js` (even through the documentation)

Fontawesome for example already has it's own implementation of an SVG renderer.

Reply:
@DRoet You are right, I'm just trying to see more possibilities and without break the previously solutions. I already talked with the @vuetifyjs/core-team about the focused aproach on `@mdi/js` and about the setting's name `mdi-svg`. The choice for the naming on settings was because we copied the paths strings from `@mdi/js` library so we have to refer them.

Both solutions were designed for a specific case, like you said, but they allow its usage by coping the inner path.

Another thing we can add there is a way to allow `<v-icon><svg ...>...</svg></v-icon>` itself.

Libraries, apps, icon packs, and others that uses or exports icons and images as svg or path:
https://www.flaticon.com/ (that one has some that doesn't end with `Z` or `z`)
https://icomoon.io/app/
https://simpleicons.org/
https://linearicons.com/free

Also, please notice that a lot of svg paths ends with `z` instead of `Z`.